src/User32/User32+DeviceOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains the <see cref="DeviceOptions"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        public enum DeviceOptions : uint
+        {
+            EDD_GET_DEVICE_INTERFACE_NAME = 0x00000001
```

Comment:
nit: add a trailing comma. I think we have a stylecop rule for this anyway.
Why? So that we can add additional enum values to this without editing the existing lines.